### PR TITLE
New exchange phemex 

### PR DESCRIPTION
**Pull request by phemex**

### CODE REVIEW

File: js/phemex.js [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+        const method = 'privateGetAccountsAccountPositions';
+        const btcResponse = await this[method] ({ 'currency': 'BTC' });
+        const usdResponse = await this[method] ({ 'currency': 'USD' });
+        const btcAccount = this.safeValue (this.parseResponse (btcResponse), 'account');
+        const btcBalanceEv = this.safeValue (btcAccount, 'accountBalanceEv');
+        const btcTotalUsedBalanceEv = this.safeValue (btcAccount, 'totalUsedBalanceEv');
+        const usdAccount = this.safeValue (this.parseResponse (usdResponse), 'account');
+        const usdBalanceEv = this.safeValue (usdAccount, 'accountBalanceEv');
+        const usdTotalUsedBalanceEv = this.safeValue (usdAccount, 'totalUsedBalanceEv');
+        return {
```
Comment (newsum) [inline js/phemex.js]: Can you update the returned balance structure?

Missing `total`, `'free', and `used` on the top-level object returned.

See here: https://github.com/ccxt/ccxt/wiki/Manual#balance-structure

Comment (kroitor) [reply js/phemex.js]: Also: https://github.com/ccxt/ccxt/wiki/Requirements )

Comment (phemex) [reply js/phemex.js]: already added total, free, and used to balance structure, please review again.

**Comment by mahapo** [comment]
@kroitor Any chance to merge this soon?

**Comment by kroitor** [comment]
@mahapo can't promise, but will do what's possible as soon as i can! thx for your patience!

**Comment by mschultheiss83** [comment]
@kroitor please merge asap

**Comment by kroitor** [comment]
@mschultheiss83 working on it, will merge it very soon, within a few days.

**Pull request merged**